docs/reference/transform/checkpoints.asciidoc

```diff
@@ -55,6 +55,47 @@ TIP: If the cluster experiences unsuitable performance degradation due to the
 {transform}, stop the {transform} and refer to <<transform-performance>>.
 
 
+[discrete]
+[[sync-field]]
+== Using the ingest timestamp for syncing the {transform}
+
+In most cases, it is strongly recommended to use the ingest timestamp of the 
+source indices for syncing the {transform}. This is the most optimal way for 
+{transforms} to be able to identify new changes. It can be set by using an 
+ingest pipeline. You can create an ingest pipeline either using the 
+<<put-pipeline-api, ingest pipeline API>> (like the example below) or via {kib} 
+under **Stack Management > Ingest Pipelines**. Use a `set` processor to set a 
+field – in the example, this is `_source.@timestamp_ingest` – and associate it 
+with the value of the ingest timestamp.
```

Comment:
I suggest to comply with ECS, and use [event.ingested](https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-ingested) instead of `_source.@timestamp_ingest`.

Reply:
Changed via [f69dc50](https://github.com/elastic/elasticsearch/pull/87650/commits/f69dc506a7c4cba95927b1d1644c8d3b7afda320).